packages/docs/.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: '🐛 Bug Report'
```

Comment:
I don't think all of these github actions/issue templates/etc defined in the `packages/docs` folder will do anything. If you plan introducing them iteratively, then IMO these are safe to remove from this PR.